elpis/engines/kaldi/objects/transcription.py

```diff
@@ -18,54 +18,61 @@
 class KaldiTranscription(BaseTranscription):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        self.audio_file_path = self.path.joinpath('audio.wav')
+        self.audio_filename = None
+        self.audio_file_path = None
         self.audio_duration = 0.0
 
     @classmethod
     def load(cls, base_path: Path):
         self = super().load(base_path)
-        self.audio_file_path = self.path.joinpath('audio.wav')
         return self
 
     def _build_spk2utt_file(self, spk_id: str, utt_id: str):
-        spk2utt_path = Path(self.path).joinpath('spk2utt')
+        spk2utt_path = self.path.joinpath('spk2utt')
         with spk2utt_path.open(mode='w') as fout:
                 fout.write(f'{spk_id} {utt_id}\n')
 
     def _build_utt2spk_file(self, utt_id: str, spk_id: str):
-        utt2spk_path = Path(self.path).joinpath('utt2spk')
+        utt2spk_path = self.path.joinpath('utt2spk')
         with utt2spk_path.open(mode='w') as fout:
                 fout.write(f'{utt_id} {spk_id}\n')
 
     def _build_segments_file(self, utt_id: str, rec_id: str, start_ms: float, stop_ms: float):
-        segments_path = Path(self.path).joinpath('segments')
+        segments_path = self.path.joinpath('segments')
         with segments_path.open(mode='w') as fout:
                 fout.write(f'{utt_id} {rec_id} {start_ms} {stop_ms}\n')
 
     def _build_wav_scp_file(self, rec_id: str, rel_audio_file_path: Path):
-        wav_scp_path = Path(self.path).joinpath('wav.scp')
+        wav_scp_path = self.path.joinpath('wav.scp')
         with wav_scp_path.open(mode='w') as fout:
                 fout.write(f'{rec_id} {rel_audio_file_path}\n')
 
+    # Write audio filename to a file so the shell scripts can read it. Maybe safer than setting ENV?
+    def _build_audio_meta(self, audio_filename: Path):
```

Comment:
Write a text file containing the audio filename, as a way to get this info in the kaldi shell scripts